src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.books.push(payload);
+    },
```

Comment:
- [OPTIONAL] I highly recommend not using the `push` method which can mutate your array. However, you can use the [Destructuring JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) method. You can find more [information here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) about it.

```suggestion
    addBook: (state, { payload }) => {
      state.books = [...state.books, payload];
    },
```